app/models/solr_document.rb

```diff
@@ -50,4 +50,10 @@ def searchworks_url
   # and Blacklight::Solr::Document#to_semantic_values
   # Recommendation: Use field names from Dublin Core
   use_extension(Blacklight::Document::DublinCore)
+
+  # Check to see if view is an image
+  # Used for georeferencing message
+  def is_image?
+    file_format&.include?('JPEG')
```

Comment:
Would you please add a test for this method, including a case when file_format is nil?